firstvoices/backend/tests/test_apis/test_mtd_data_api.py

```diff
@@ -129,3 +132,26 @@ def test_data_integrity_build_and_score_failed(self, status):
         )
         assert response.status_code == 200
         assert response.data[0]["status"] == status
+
+    @pytest.mark.django_db
+    def test_etag_and_last_modified(self):
+        self.user = factories.get_superadmin()
+        self.client.force_authenticate(user=self.user)
```

Comment:
Minor: I don't think this request needs to be authenticated at all for testing, since it's available to guests